test/visual-diff/visual-tests

```diff
@@ -144,6 +144,11 @@
       "name": "Video rotate-to-fullscreen",
       "loading_complete_css": ["video.i-amphtml-replaced-content"]
     },
+    {
+      "url": "examples/visual-tests/amp-sticky-ad/amp-sticky-ad-wrapper.html",
+      "name": "AMP Sticky Ad",
+      "loading_complete_css": ["ready-for-visual-diff"]
```

Comment:
I believe this would not work, because the page that gets tested is the wrapper, and the browser will be looking for the CSS class in that page, not inside the iframe. How about sending a message from inside the iframe to the container document, and have the container document add the `ready-for-visual-diff` class to the iframe on message receipt?

nit: it should be `.ready-for-visual-diff`, i.e. preceded by a dot

Reply:
Ah! Good point! Switched to postMessage.